src/tools/data.js

```diff
@@ -142,14 +142,17 @@ const recurseExtract = (obj, matcher) => {
 const _IGNORE = {};
 
 // Flatten a nested object.
-const flattenPaths = (data, sep = '.') => {
```

Comment:
A little refactor to tidy up the function. We never call this function with a different separator, so let's just only use `'.'`. If we do someday flatten objects with a different separator, it will be easy to add since we're now using an options object.

Other changes are using defaults and better native methods like `Object.entries` and `.forEach`. `.map` to me implies we're not mutating an outside objects and will be using the returned array. `.forEach` is a better fit for what we're doing.